openaerostruct/aerodynamics/viscous_drag.py

```diff
@@ -72,7 +72,7 @@ def compute(self, inputs, outputs):
             S_ref = inputs["S_ref"]
             widths = inputs["widths"]
             lengths = inputs["lengths"]
-            cos_sweep = inputs["cos_sweep"] / widths
+            cos_sweep = inputs["widths"] / inputs["lengths_spanwise"]

```

Comment:
Maybe I'm missing something, but I no longer see the bug here? Won't this have the same value as before?

Reply:
The bug in viscous drag computation was coming from the `widths` variable, which was the diagonally stretched spanwise width. The viscous drag component had been using `widths` to compute the panel area, hence the overestimation of viscous drag.
In other words, the bug was because of the inconsistency of the `widths` definition between the viscous drag component and geometry component (so we could say the bug was not in viscous drag itself, but it was about geometry variable naming).

Now I modified the OM variable names, and the `widths` variable is now actually the width along the y-axis (or zero sweep direction). The actual bug fix was done in `aerodynamics/geometry.py` where I changed the definition of the `widths` variable.

And yes, the `cos_sweep` python variable has the same value as before.

The same applies to the CM and sectional CL fixes; those bugs were fixed by changing the definition of `widths` in the geometry component. That's why you don't see the code changes in the CM and sectional CL components.

Reply:
Ah I see it now, thanks for the help. It was a bit sneaky that by changing the variable name it fixed something later that you didn't edit 😏